# Question 1

<img src="CapteurCylindroParabolique.png">

Let's consider a cylindro-parabolic collector with an opening width <b>D=3 m</b>, 
a length <b>L=30 m</b> and a surface reflectivity $\rho^{*}=0,86$.

The absorbing surface is a black linear receiver tube with an absorptivity $\alpha^*=0,94$
and a diameter <b>d=5 cm</b> which intercepts a fraction $\gamma=0,7$ of the reflected solar irradiation. 

The fluid circulating into the tube is oil and it has a specific heat $c_p=2 \, kJ \,  kg^{-1} \,  K^{-1}$
and a mas flow rate $\dot{m} = 1 \, kg/s$. 

The oil temperature at the supply of the tube is <b>300  °C</b>. 

The external temperature is <b>25  °C</b> and the convective heat transfer coefficient with the ambient air is equal to $7 \, W \, m^{-2} \, K^{-1}$.

The solar irradiance is supposed to be equal to <b>$ 800 \, W \, m^{-2}$</b>.

<b>Determine:</b>
 - The oil temperature at the exhaust (<b>T_f</b>).
   
 - The thermal power exchanged with the oil.
   
 - The efficiency of the collectors
   
<b>Answers</b>

$T_f = 315.7 \,  °C$ , $\dot{Q} =31,4 \, kW$ , $\eta_{coll} = 43,6 \, \% $

In [1]:
from numpy import *
from math import exp

D = 3
L = 30
rho_star = 0.86
alpha_star = 0.94
d=0.05
gamma = 0.7
c_p = 2000
T_0 = 300
m_dot = 1
T_a = 25
k=7
S_star = 800

a = -k*pi*d/(m_dot*c_p)
b = S_star*D*rho_star*gamma*alpha_star/(m_dot*c_p)+k*pi*d*T_a/(m_dot*c_p)
c = T_0 - ((S_star*D*rho_star*gamma*alpha_star+k*pi*d*T_a)/(k*pi*d))

T_f = -b/a + c*exp(a*L)

Q_dot = m_dot*c_p*(T_f-T_0)

eta_c = 1-(T_a+273.15)/(T_f+273.15)

eta_coll = Q_dot/(S_star*L*D)

eta_g = eta_coll*eta_c/2

print "T_f =", T_f, "°C", "," ,"Q_dot =", Q_dot, "W","," ,"eta_coll =", eta_coll,"eta_carnot =", eta_c,"," ,"eta_g =", eta_g

T_f = 315.706126154 °C , Q_dot = 31412.2523085 W , eta_coll = 0.436281282063 eta_carnot = 0.493679378107 , eta_g = 0.107691536004


# Question 2

The solar collectors described in the exercise 1 are connected to a Rankine cycle to produce electricity. 

For that cycle, consider that the heat source temperature is equal to the oil temperature at the exhaust of the collectors and the heat sink temperature is equal to 25°C.

Determine:
- The carnot efficiency of the rankine cycle.
- The global efficiency of the system (collectors+rankine cycle) if the energy conversion efficiency (heat to electricity) is equal to 50 % of the carnot efficiency.

<b>Answers</b>: $\eta_{carnot}$ = 49.37 % & $\eta_{global}$ = 10.8 %

# Question 3

<img src="TourSolaire.png" >

Let's consider the PS10 Solar Power Plant in Spain near Seville. This is a concentrating solar power tower with a nominal electrical power of 11 MW. 

The main components of the power plant are the following:
- 624 large sun-tracking mirrors (called heliostats)
- A 115 meter high tower
- A solar receiver (the solar boiler) situated at the top of the tower
- A storage to store thermal energy
- A classical rankine cycle to produce electricity

The sun-tracking mirrors focus the sunlight on a receiver situated at the top of the tower. The receiver converts the solar radiation into heat. A heat-transfer fluid (hot air or molten salt for example) is then used to transport the heat from the absorber to a steam generator, to produce superheated water steam. This water steam drives a turbine and an electrical generator to produce electricity.


The characteristics of the power plant are the following:
    
<ul>

<li> The number of heliostats is $N_h=624$ </li>
<li> The surface area of one heliostat is $S_h= 120 \, m^2$ </li>
<li> The solar irradiance is $I^*=750 \, W/m^2$ </li>
<li> The effective concentration of the installation is $C_e^* = 500$ </li>
<li> The solar boiler temperature is $T_{ch}=275°C$ </li>
<li> The ambiant temperature is $T_a=35°C$ </li>
<li> The boiler absorptivity is $\alpha^*_{ch}=0,7$ </li>
<li> The boiler emissivity factor is $\varepsilon_{ch}=0,85$ </li>
<li> The efficiency of the rankine cycle is equal to $ 66 \, \%$ of the carnot efficiency (using the same heat sink and heat source temperatures).</li>

</ul>  

Determine the efficiency and the electrical output of the whole system.

<b>Answers</b>

$ P_{mec}  =  11,3 \, MW$   &   $\eta_{global} =  20,12 \, \%$.

In [5]:
N_h = 624
S_h = 120
I_star = 750
C_star_e = 500
T_ch = 275+273.15
alpha_star_ch = 0.7
epsilon_ch = 0.85
sigma_0 = 5.6703*10**(-8)
T_a = 35+273.15

P_tot = N_h*S_h*I_star/1000000

eta_ch = alpha_star_ch*(1-(epsilon_ch*sigma_0*(T_ch**4-T_a**4)/(alpha_star_ch*C_star_e*I_star)))
eta_c = 1-(T_a)/(T_ch) 

eta_g = eta_ch*eta_c*0.66

print P_tot, eta_ch, eta_c, eta_g


56 0.689555327339 0.437836358661 0.199262179788
